x-pack/plugins/security_solution/public/common/components/event_details/response_actions_view.tsx

```diff
@@ -36,61 +65,69 @@ export const useResponseActionsView = <T extends object = JSX.Element>({
   ecsData?: Ecs | null;
   rawEventData: SearchHit | undefined;
 }): EuiTabbedContentTab | undefined => {
+  const viewData = useMemo(
+    () => ({
+      id: EventsViewType.responseActionsView,
+      'data-test-subj': 'responseActionsViewTab',
+      name: i18n.RESPONSE_ACTIONS_VIEW,
+    }),
+    []
+  );
```

Comment:
You can lift this `const` to outside the hook without the need of an extra `useMemo`, since it's static.

Reply:
Yeah I tried already, and unfortunately there is an issue with `EventsViewType`.

```ReferenceError: Cannot access 'EventsViewType' before initialization``` 
Do you mind if I leave it wrapper with useMemo?

Reply:
As a runtime error?

Reply:
Yes